src/token/history/history.resolver.ts

```diff
@@ -0,0 +1,58 @@
+import { Args, Int, Mutation, Resolver } from '@nestjs/graphql'
+import { Prisma } from '@prisma/client'
+import { Roles } from '../../auth/decorators/roles.decorator'
+import { TokenHistory } from '../../token/history/entities/token-history.entity'
+import { HistoryTokenService } from '../../token/history/history.service'
+
+@Resolver(() => TokenHistory)
+export class TokenResolver {
+    constructor(private readonly historyTokenService: HistoryTokenService) {}
+
+    @Mutation(() => TokenHistory, { nullable: true })
+    async getMyRoomById(@Args('id', { type: () => Int }) id: number):Promise<TokenHistory>{
```

Comment:
Need to rename. And when you use `My` in method name you must include decorator `@GetCurrentUserId()`. And filter all data by `userId`